net/src/ipc/transport_wss/mod.rs

```diff
@@ -271,60 +271,56 @@ impl<T: Read + Write + std::fmt::Debug> TransportWss<T> {
                     .expect("failed to build TlsConnector");
                 info.stateful_socket =
                     self.priv_tls_handshake(connector.connect(info.url.as_str(), socket))?;
-                return Ok(());
+                Ok(())
             }
             WssStreamState::TlsMidHandshake(socket) => {
                 info.stateful_socket = self.priv_tls_handshake(socket.handshake())?;
-                return Ok(());
+                Ok(())
             }
             WssStreamState::TlsReady(socket) => {
                 info.last_msg = std::time::Instant::now();
                 *did_work = true;
                 info.stateful_socket = self
                     .priv_ws_handshake(&info.id, tungstenite::client(info.url.clone(), socket))?;
-                return Ok(());
+                Ok(())
             }
             WssStreamState::WssMidHandshake(socket) => {
                 info.stateful_socket = self.priv_ws_handshake(&info.id, socket.handshake())?;
-                return Ok(());
+                Ok(())
             }
             WssStreamState::Ready(mut socket) => {
                 let msgs: Vec<Vec<u8>> = info.send_queue.drain(..).collect();
                 for msg in msgs {
-                    if let Err(e) = socket.write_message(tungstenite::Message::Binary(msg)) {
-                        return Err(e.into());
-                    }
+                    socket.write_message(tungstenite::Message::Binary(msg))?;
```

Comment:
uhh... this logic is wrong (it was wrong before your change too).

Can you add a comment or something? since it drains them, if an error happens, all the remaining messages will be lost.